### PR TITLE
Add information about GraphQL merging schema files

### PR DESCRIPTION
**Pull request by MastalerzKamil**
CHANGES:
- [x] add info about merging schema

### CODE REVIEW

**Review by arkjel — APPROVED** [review]

**Review by krzyjel — APPROVED** [review]

File: docusaurus/docs/getting-started.mdx [hunk 1]
```diff
@@ -71,17 +71,18 @@ Do you want add more identity providers? (each provider requires an active devel
 1. Go to `cloud` directory and fill credentials in `.env` file.
  Required environmental variables are listed in
 `.env.example`
-2. When the project will be generated, go to cloud and use
+2. When the project will be generated, go to cloud and use. The command will merge all `.graphql` into `lib/api/schema.graphql`
```
Comment (arkjel) [inline docusaurus/docs/getting-started.mdx:74]:
```suggestion
2. When the project will be generated, go to the `cloud` directory. The command will merge all `.graphql` files into `lib/api/schema.graphql`
```

**Review by arkjel — CHANGES_REQUESTED** [review]
Please update that sentence

File: docusaurus/docs/getting-started.mdx (hunk 1 above)
Comment (arkjel) [reply docusaurus/docs/getting-started.mdx:74]: and wait... where is that comand?

File: docusaurus/docs/getting-started.mdx [hunk 2]
```diff
@@ -71,17 +71,18 @@ Do you want add more identity providers? (each provider requires an active devel
 1. Go to `cloud` directory and fill credentials in `.env` file.
  Required environmental variables are listed in
 `.env.example`
-2. When the project will be generated, go to cloud and use
+2. When the project will be generated, go to the `cloud` directory. The command will merge all `.graphql` files into `lib/api/schema.graphql`
```
Comment (arkjel) [inline docusaurus/docs/getting-started.mdx:74]: Lack of the mentioned comment

**Review by arkjel — APPROVED** [review]

**Pull request merged**